css-three-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+
+body {
+  font-family: Lato;
+  font-style: normal;
+  font-weight: 400;
+  background-color: rgb(23, 140, 212);
+  color: linen;
+}
+
+.row {
+  display: flex;
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-third {
+  width: calc(100% / 3);
+}
+
+.pictures {
+  height: auto;
+  width: 100%;
+  padding: 25px;
+}
+
+.background {
+  background-color: white;
+  border-radius: 50%;
+  height: auto;
+  width: 25%;
+  margin: auto;
+}
+
+.blocks {
+  justify-content: center;
+}
+
+h2 {
+  text-align: center;
+  justify-content: center;
+  align-content: center;
+  letter-spacing: 0.1rem;
+  font-size: 2rem;
+}
+
+.words {
+  color: whitesmoke;
+  width: (100% / 3);
```

Comment:
This would need the `calc` css function to do anything, I imagine this is doing nothing right now 🤔